### ISSUE TITLE
Usage with fastify-cli

### ISSUE THREAD

**Issue by sinedied** [autotelic/fastify-opentelemetry#58]
I tried to setup this package in a fastify plugin, but I get this error:

```
[App] ApplicationInsights:@opentelemetry/instrumentation-fastify [
[App]   'Module fastify has been loaded before @opentelemetry/instrumentation-fastify so it might not work, please initialize it before requiring fastify'
[App] ]
```

I'm assuming that the setup should all be done before fastify is even loaded, but I'm using the fastify-cli so it that even possible? My project was generated from the fastify-cli template.

**Comment by HW13** [comment]
Hi @sinedied,

Have you tried using fastify-cli's `--require` option ([docs](https://github.com/fastify/fastify-cli#options)) to preload your OpenTelemetry configuration file?

```sh
fastify start index.js --require <path to config>
```

**Comment by sinedied** [comment]
Yes, that's what I tried after some reading, but I'm stuck as I did not found a way to make it work within an ESM project: https://github.com/fastify/fastify-cli/issues/671

**Comment by HW13** [comment]
Node (v18.18.0) has an experimental [`--import`](https://nodejs.org/api/cli.html#--importmodule) option for preloading ESM files. This could be set with the `NODE_OPTIONS` environment variable before starting the server.

```sh
NODE_OPTIONS="--import=path_to_config" npm start
```

**Comment by HW13** [comment]
Hey @sinedied

I'm going to close this as there isn't much we can do on our end to resolve this issue. If you find a solution or have any more questions please feel free to post it here or create a new issue. Thanks!

**Comment by sinedied** [comment]
@HW13 No worries, and thanks for suggesting using `--import`. I have yet to try it, but it sounds like a good workaround in the meantime ES modules are supported by the fastify CLI.

**Comment by wlinna** [comment]
At least for me, fastify cli's `--import` doesn't help while adding `NODE_OPTIONS=--import=./dist/earlyconfig.js` to my `.env` does.
